### ISSUE TITLE
[Bug]: Lyrics not highlighted for long enough (for word by word lyrics)

### ISSUE THREAD

**Issue by xXTgamerXx** [ciderapp/Cider-2#461]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ciderapp/Sabiiro-BugTracker/issues) for a bug report that matches the one I want to file, without success.
- [X] I have read the [troublshooting FAQs](https://cider.gitbook.io/welcome-to-gitbook/support/faqs) and none solved my issue.

### What operating system are you using?

Windows

### Build Details

Built on 18/9/2023, 1:46:34 AM

### Operating System Version

Windows 11 Pro Insider Preview 22H2 Build 23526.1000

### Describe the Bug

When a song supports word by word lyrics, the lyrics line does not stay highlighted for long enough.
E.g. Comparison between Cider (a) and Apple Music Web (b)
a) 
![Cider](https://github.com/ciderapp/Cider-2/assets/72494265/ddc8863d-a540-4efa-ab65-26d6c8c9a6ff)
b) 
![Apple Music](https://github.com/ciderapp/Cider-2/assets/72494265/2b3cad81-beb2-40dc-b76f-397f8c70e187)
- In (b), the lyrics line stays highlighted till the next line starts, which is helpful for reading the lyrics, however in (a) the highlight stops once the last word is finished.

### Steps to Reproduce

Open lyrics for a song that has support for the word by word lyrics.

### Build timestamp

1702055292843

**Comment by coredev-uk** [comment]
This issue has been closed to consolidate the issue tracker for Cider 2.3.1 and later issues. If you believe this issue is still relevant, please open a new issue.
